<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportForm</class>
 <widget class="QWidget" name="ReportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Medical Report</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0" colspan="6">
     <widget class="QTextEdit" name="notesEdit"/>
    </item>
    <item row="2" column="2" colspan="4">
     <widget class="QLineEdit" name="symptomsEdit"/>
    </item>
    <item row="4" column="2" colspan="4">
     <widget class="QLineEdit" name="treatmentEdit"/>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="viewReportsButton">
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="4">
     <widget class="QLineEdit" name="patientDEdit"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Symptoms:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Diagnosis:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Patient ID:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Treatment:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Notes:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="reportBackButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="4">
     <widget class="QLineEdit" name="diagnosisEdit"/>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QPushButton" name="addReportButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="4">
     <widget class="QDateEdit" name="reportDateEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
